src/chess/chess-arbiter.js

```diff
@@ -0,0 +1,273 @@
+import * as PositioningUtils from "./positioning-utils";
+import * as GameNetwork from "./game-network";
+import * as Chess from "chess.js";
+
+AFRAME.registerSystem("chess-arbiter", {
+  init() {
+    this.tick = AFRAME.utils.throttleTick(this.tick, 300, this);
+    this.state = this.sceneEl.systems.state.state;
+    this.chessGame = this.sceneEl.querySelector("a-entity[chess-game]");
+    this.sceneEl.addEventListener("chess-command", ev => {
+      const command = ev.detail[0];
+      this.handleChatCommand(command);
+    });
+    this.resetGame();
+    GameNetwork.setupNetwork(this.sceneEl);
+  },
+
+  tick() {
+    if (this.state.imPlaying) {
+      const pieces = this.sceneEl.querySelector("a-entity[chess-set]").children;
+      for (const piece of pieces) {
+        if (piece.metadata) {
+          this.interactionHandler(piece);
+        }
+      }
+    }
+  },
+
+  resetGame() {
+    this.chessEngine = new Chess();
+  },
+
+  handleChatCommand(command) {
+    const id = GameNetwork.getMyId();
+    const profile = window.APP.store.state.profile;
+    switch (command) {
+      case "play":
+        const color = ev.detail[1];
```

Comment:
Linter flagged this as `ev is not defined` and sure enough it errors in console when trying this command

Reply:
This will be fixed in the chess-refactor-fixes PR awaiting review.